frontend/packages/kubevirt-plugin/integration-tests/tests/utils/scripts/drag-drop.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Drag and Drop function implemented in pure JavaScript
+ * Used due to chromedriver issues with mouse movement and
+ * browser.actions().dragAndDrop(src, dst).perform() not working.
+ * Source: https://gist.github.com/florentbr/60ef7cb8d9b1ae690cafc82aad52da73#file-drag-drop-js
+ */
+export const dragAndDrop = `
```

Comment:
it is a compromise (:

Reply:
@suomiy  Oh dear, I missed you have approved this change :-O, wanted to split this PR into two parts to deal with the drag and drop later. I have reverted the change to the previous state, would you please re-lgtm?